pandas/tests/io/test_pytables.py

```diff
@@ -5167,3 +5168,75 @@ def test_dst_transitions(self):
                 store.append('df', df)
                 result = store.select('df')
                 assert_frame_equal(result, df)
+
+
+class TestReadPyTablesHDF5(Base):
+    """
+    A group of tests which covers reading HDF5 files written by plain PyTables
+    (not written by pandas).
+    """
+
+    def _create_simple_hdf5_file_with_pytables(self):
+
+        table_schema = {
+            'c0': tables.Time64Col(pos=0),
+            'c1': tables.StringCol(5, pos=1),
+            'c2': tables.UInt32Col(pos=2),
+        }
+
+        t0 = time.time()
+
+        testsamples = [
+            {'c0': t0, 'c1': 'aaaaa', 'c2': 1},
+            {'c0': t0 + 1, 'c1': 'bbbbb', 'c2': 2},
+            {'c0': t0 + 2, 'c1': 'ccccc', 'c2': 10**5},
+            {'c0': t0 + 3, 'c1': 'ddddd', 'c2': 4294967295},
+        ]
+
+        # This returns a path and does not open the file.
+        tmpfilepath = create_tempfile(self.path)
```

Comment:
so this should just be a fixture

Reply:
:+1: 

I now turned the entire thing into a module-scoped fixture, providing the path to the HDF5 file and details about its contents.